include/nw4r/ut/Color.h

```diff
@@ -7,18 +7,31 @@ namespace ut {
 
 struct Color : public _GXColor {
 public:
+    Color() {
+        *this = 0xFFFFFFFF;
+    }
+
+    Color(const Color &rhs) {
```

Comment:
Does the copy constructor need to be explicitly defined to match? The compiler should be able to generate it automatically.